src/datepicker/datepicker-input.spec.ts

```diff
@@ -0,0 +1,220 @@
+import {TestBed, ComponentFixture, fakeAsync, tick} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {createGenericTestComponent} from '../test/common';
+
+import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+
+import {NgbDatepickerModule} from './datepicker.module';
+import {NgbInputDatepicker} from './datepicker-input';
```

Comment:
Shouldn't the directive be named NgbInputDatepicker, or the file be named input-datepicker?